input/docs/integrations/editors/rider/run-configurations.md

```diff
@@ -0,0 +1,125 @@
+Order: 20
+Title: Run configurations
```

Comment:
I would suggest to call it `Rider run configuration` which would improve search results

```suggestion
Title: Rider run configurations
```